### PR TITLE
Endoc 194 api docs

### PR DESCRIPTION
**Pull request by nshaw**
Improve the API docs

### CODE REVIEW

File: vuepress/docs/next/docs/reference/entando-apis.md [hunk 1]
```diff
@@ -5,28 +5,53 @@ sidebarDepth: 2
...
+### Enable or disable the Swagger UI in a running container
+
+The Swagger UI can be enabled or disabled in a running container by modifying the SPRING_PROFILES_ACTIVE environment variable for the entando-de-app container. 
+
+1. Edit the deployment. The name may be different outside of a quickstart environment.
+```
+sudo kubectl -n entando edit deployment/quickstart-server-deployment
+```
+
+2. Scale the deployment `spec.replicas` to 0 before updating the deployment. This helps avoid database errors that can happen on an immediate restart after the profile is changed. Save the edit to apply the change.
```
Comment (joewhite101) [inline vuepress/docs/next/docs/reference/entando-apis.md:21]: This is only true/required in an environment with in memory databases. Might be worth clarifying.

File: vuepress/docs/next/docs/reference/entando-apis.md [hunk 2]
```diff
@@ -5,28 +5,53 @@ sidebarDepth: 2
...
+
+4a. To enable the swagger UI, add the SPRING_PROFILES_ACTIVE environment variable, if it's missing, or add `swagger` to its comma-delimited list.
+
+```
+        - name: SPRING_PROFILES_ACTIVE
+          value: default,swagger
+```
+4b. To disable the swagger UI, remove `swagger` from the value.
+
+5. Reset the deployment `spec.replicas` back to 1.
```
Comment (joewhite101) [inline vuepress/docs/next/docs/reference/entando-apis.md:33]: Same as above

**Review by joewhite101 — APPROVED** [review]
One note.

File: vuepress/docs/next/docs/reference/entando-apis.md (hunk 1 above)
Comment (nshaw) [reply vuepress/docs/next/docs/reference/entando-apis.md]: Good to know. Updated.

File: vuepress/docs/next/docs/reference/entando-apis.md (hunk 2 above)
Comment (nshaw) [reply vuepress/docs/next/docs/reference/entando-apis.md]: Done

**Pull request merged**